index.js

```diff
@@ -35,92 +56,29 @@ const sessionsConfig = {
 //======session configuration====//
 
 server.use(session(sessionsConfig));
-server.use(express.json());
-server.use(cors());
+
+server.use("/api", login);
 
 //========middleware==========//
-function auth(req, res, next) {
-  if (req.session && req.session.username) {
-    next();
-  } else {
-    res.status(401).json({ Error: "You shall not pass!!" });
+function errorHandler(err, req, res, next) {
+  console.error(err);
```

Comment:
This `errorHandler` function is new to me, so I'm legitimately curious. How does it work? When does express call it?

Reply:
The errorHandler is just middle ware  for when the get(/download) doesn't work